src/map.cpp

```diff
@@ -1147,12 +1147,15 @@ bool Map::isBlockOccluded(MapBlock *block, v3s16 cam_pos_nodes)
 
 	v3s16 pos_blockcenter = block->getPosRelative() + (MAP_BLOCKSIZE / 2);
 
-	// Starting step size, value between 1m and sqrt(3)m
-	float step = BS * 1.2f;
+	// Starting step size, slightly > sqrt(3),
+	// so that each block is hit at most once.
```

Comment:
this initially confused me, it should say "so that each node is hit at most once"

Reply:
Sorry, yes, each node. I'll update the comment in the next round.